bounties_api/std_bounties/urls.py

```diff
@@ -15,6 +15,7 @@
 urlpatterns = [
     url(r'^leaderboard/issuer/$', views.LeaderboardIssuer.as_view()),
     url(r'^leaderboard/fulfiller/$', views.LeaderboardFulfiller.as_view()),
+    url(r'^fulfiller/(?P<address>\d+)', views.UserProfile.as_view()),
```

Comment:
@iamonuwa why are you adding a url here? UserProfile already exists as an endpoint. It is in the User app.

Reply:
No need to add this extra endpoint.

Reply:
Will check again.

Reply:
@villanuevawill I'll remove this as method from bounties views?